src/scope/scope.ts

```diff
@@ -277,10 +277,13 @@ export default class Scope {
     // Make sure to not start the loader if there are no components to build
     if (components && components.length) {
       loader.start(BEFORE_RUNNING_BUILD);
+      if (components.length > 1) loader.stopAndPersist();
```

Comment:
@JoshK2 why do you need this `stopAndPersist` here?

Reply:
I want to show a message when the user is building more than one component.
I update it to this: `loader.stopAndPersist({ text: `${BEFORE_RUNNING_BUILD}...` });`
So, for example, this is what happens when you build one or more:
![image](https://user-images.githubusercontent.com/35892475/68202357-3dfe5480-ffcc-11e9-8b5f-db1d8da2f0e8.png)

The `stopAndPersist` function stops the loader and prints the current text or new one if given.

Reply:
@JoshK2 The problem with this stop and persist is that if we will add --json option to build it will cause issues.
I just saw that by default it will write to the stderr, https://github.com/sindresorhus/ora#stream
So it might be fine with `--json` but we should make sure it doesn't cause issues in bit.dev in the ci tab.